codebasin.py

```diff
@@ -131,6 +132,13 @@ def main():
     logging.getLogger("codebasin").setLevel(
         max(1, logging.WARNING - 10 * (args.verbose - args.quiet)),
     )
+
+    # Warnings for deprecated functionality with no planned replacement.
+    if args.batchmode:
```

Comment:
I think throwing [`DeprecationWarning`](https://docs.python.org/3/library/exceptions.html#DeprecationWarning) is probably more fitting in these cases?

Reply:
Good catch.  I keep forgetting what the default for `warning.warn` is...  Fixed in e835aeadd72e6b57d36b6def88af42d333533c29.